lib/msal-browser/FAQ.md

```diff
@@ -328,12 +328,15 @@ Token lifetimes are 1 hour and the session lifetime is 24 hours. This means that
 
 Please see the documentation on [Tenancy in Azure Active Directory](https://docs.microsoft.com/azure/active-directory/develop/single-and-multi-tenant-apps#who-can-sign-in-to-your-app)
 
-# Scopes & Resources
+# Scopes & Resources - [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/ellymakuba-patch-1/lib/msal-angular/docs/v2-docs/msal-interceptor.md#protected-resource-map)
```

Comment:
Sorry, I mean link to this FAQ from the msal-interceptor.md page.

Reply:
I have addressed this comment, please confirm it's resolved.